Slash/Utility/Comments/Comments.pm

```diff
@@ -1300,12 +896,9 @@ sub displayThread {
 	my $hidden = my $skipped = 0;
 	my $return = '';
 
-	my $discussion2 = discussion2($user);
-	my $threshold = -1; #$discussion2 && defined $user->{d2_threshold} ? $user->{d2_threshold} : $user->{threshold};
-	my $highlightthresh = -1; #$discussion2 && defined $user->{d2_highlightthresh} ? $user->{d2_highlightthresh} : $user->{highlightthresh};
-	$highlightthresh = $threshold if $highlightthresh < $threshold;
+	my $threshold = -1;
+	my $highlightthresh = -1;
```

Comment:
These should not be set if we do not use them elsewhere to change the status of any comment or to drop one.  I only set them to -1 to force the old code to give me all of the comments.